### DISCUSSION TITLE
Dicas de produtividade

### DISCUSSION THREAD

**Discussion by bernardodiasc** [frontendbr/forum#545] [Dicas]
Sabe aquela feature que voce implementou e vai mandar num PR? As vezes precisa de uma pequena demonstração, ou várias, e tem diversas formas de fazer isso, vou mostrar uma.

O plano free do [CloudApp](https://www.getcloudapp.com/) grava GIFs de até 15 segundos. Se for o caso de gravar vários mini-demos o processo todo fica bem prático.

## Veja como funciona

![screen recording 2017-04-20 at 04 46 pm](https://cloud.githubusercontent.com/assets/131859/25249784/8bd5eb7e-25e9-11e7-8841-f91cc82063a2.gif)

---

Quem quiser compartilhar mais coisinhas úteis do dia-a-dia, manda ver :)

**Comment by calaca** [comment]
Tava achando maravilhoso aí vi que não tem uma versão para Linux. 😞

**Comment by ericdouglas** [comment]
# Dica 1
@calaca para gravar um vídeo no Linux é bem simples, use o programa [SimpleScreenRecorder](http://www.maartenbaert.be/simplescreenrecorder/). A dica 3 tem uma opção para gravar gif.

# Dica 2
Pra quem quiser tirar um print da tela e selecionar a parte da tela que é para ficar na imagem, basta usar o programa [shutter](http://shutter-project.org/preview/screenshots/).

Depois você pode criar um atalho no OS para quando apertar `alt + s` o programa abrir, simulando a funcionalidado que tem no Mac.

Configure shortcut:

Settings > keyboard > shortcuts tab > Add custom shortcuts

Define <kbd>Alt+S</kbd> for shortcut. 

# Dica 3
Para gravar gif, tem o `byzanz`:

**Usage**

```
sudo apt-get install byzanz
```

When its installed you can run it in a terminal.

This is a small example I did just now in a vbox with

```
byzanz-record --duration=30 --x=0 --y=0 --width=1366 --height=740 out.gif
```

# Outros

Mais alguns programas úteis que uso no linux [aqui](https://gist.github.com/ericdouglas/8e943c12046ccc2eada2).

**Comment by calaca** [comment]
@ericdouglas Valeu! Aproveitando o espaço, experimente o [screen cloud](https://screencloud.net/) para tirar prints ao invés do shutter, ele tem umas funcionalidades parecidas com o [lightshot](https://app.prntscr.com/pt-br/) (windows e mac).

**Comment by bernardodiasc** [comment]
Uso muito o Nimbus pelo Chrome tambem https://chrome.google.com/webstore/detail/nimbus-screenshot-screen/bpconcjcammlapcogcnnelfmaeghhagj?hl=en

**Comment by didiraja** [comment]
Um plugin do **Brackets** que me poupou um tempo legal foi o [Starter Bootstrap Template](https://github.com/JTruax/bootstrap-starter-template).

Com seu uso, acabou me induzindo a montar o que na verdade é minha grande dica nesse tópico:
### **Monte seu próprio kit de iniciação de projetos já com arquivos, libs e códigos que eu sempre usa em seus projetos.** 

Você não precisa toda vez digitar <html>, <body> ou colar os caminho do bootstrap/font-awesome/angularjs.
